tests/standalone_tests/test_monet_db_facade.py

```diff
@@ -0,0 +1,107 @@
+from unittest.mock import patch
+
+import pytest
+
+from mipengine import AttrDict
+from mipengine.node.monetdb_interface.monet_db_facade import db_execute_and_fetchall
+from mipengine.node.node_logger import init_logger
+from tests.standalone_tests.conftest import COMMON_IP
+from tests.standalone_tests.conftest import MONETDB_LOCALNODETMP_NAME
+from tests.standalone_tests.conftest import MONETDB_LOCALNODETMP_PORT
+from tests.standalone_tests.conftest import _restart_monetdb_container
+
+
+@pytest.fixture(scope="module", autouse=True)
+def mock_current_task():
+    current_task = AttrDict({"request": {"id": "1234"}})
+
+    with patch(
+        "mipengine.node.node_logger.node_config",
+        AttrDict(
+            {
+                "log_level": "DEBUG",
+                "role": "localnode",
+                "identifier": "localnodetmp",
+            },
+        ),
+    ), patch(
+        "mipengine.node.node_logger.task_loggers",
+        {"1234": init_logger("1234")},
+    ), patch(
+        "mipengine.node.node_logger.current_task",
+        current_task,
+    ):
+        yield
+
+    return current_task
```

Comment:
nit: `current_task` doesn't need to be returned. Also, what does` current_task` mean in this context? Maybe rename the method to something like `patch_node_logger` ?